deploy/helm/sumologic/templates/configmap.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ template "sumologic.fullname" . }}-fluentd-logs
+  name: {{ template "sumologic.metadata.name.logs" . }}
```

Comment:
I think it will be more meaningfull if we use `configmap` at the end
so the full name will be:
`sumologic.metadata.name.logs.configmap`

Given that Helm is `stringly typed` it will be easier to reason about what is being used where.

Reply:
+1